gcd/config/index.yaml

```diff
@@ -15,7 +17,16 @@ indexes:
   # The Datastore kind represents the Protobuf type name of the Aggregate state. Both the property
   # names and the index directions should be kept unchanged.
 
-  - kind: spine.test.aggregate.Project
+  - kind: myapp.example.MyFirstAggregate
```

Comment:
Please have a comment following this line that would encourage the user to replace only this line with an aggregate type name. And only it.